data/i18n/stats.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
I guess the change to interpreter is just to test, but should probably be kept as just ```python```, since with the below change the script is valid for 2 or 3, and keeping the interpreter as ```python``` will allow it to continue working on any other host that has only 2?